pytest_tests/testsuites/acl/test_eacl.py

```diff
@@ -252,436 +252,6 @@ def test_extended_acl_deny_replication(
                 storage_nodes,
             )
 
-    @allure.title("Testcase to validate NeoFS system operations with extended ACL")
-    def test_extended_actions_system(self, wallets, eacl_container_with_objects):
-        user_wallet = wallets.get_wallet()
-        ir_wallet, storage_wallet = wallets.get_wallets_list(role=EACLRole.SYSTEM)[:2]
-
-        cid, object_oids, file_path = eacl_container_with_objects
-        endpoint = self.cluster.default_rpc_endpoint
-
-        with allure.step("Check IR and STORAGE rules compliance"):
-            assert not can_put_object(
-                ir_wallet.wallet_path,
-                cid,
-                file_path,
-                shell=self.shell,
-                cluster=self.cluster,
-                wallet_config=ir_wallet.config_path,
-            )
-            assert can_put_object(
-                storage_wallet.wallet_path,
-                cid,
-                file_path,
-                shell=self.shell,
-                cluster=self.cluster,
-                wallet_config=storage_wallet.config_path,
-            )
```

Comment:
For example, this pair perfectly makes sense as a test for basic ACL, SN should be able to PUT, while IR should not.

Reply:
> For example, this pair perfectly makes sense as a test for basic ACL, SN should be able to PUT, while IR should not.

Fixed